### DISCUSSION TITLE
performance tests on microcks

### DISCUSSION THREAD

**Discussion by darespa12** [microcks/microcks#1052] [Community]
Hello, greetings to everyone, we are very excited about microcks! We are doing tests with microcks since we have a good number of APIs, we have microcks mounted on kubernetes with a single microcks pod with 4000m of cpu and 2Gi of ram, with We are testing the above from jmeter, but we are not achieving the desired tps, the question we have is, do they have the tps value that microcks achieves in this context?

**Comment by lbroudoux** [comment]
Hey @darespa12 

Welcome here! We had some performance tuning in the past and I found this interesting conversations with TPS figures: https://github.com/microcks/microcks/issues/540#issuecomment-1004897340

Let me know if it helps. On a related topic, we are currently working on finalizing the OpenTelemetry support and building a benchmark suite for Microcks. This is definitely a topic we want to investigate with your help. 

Cheers,

**Reply by lbroudoux** [reply]
We did have some documentation on this topic here: https://microcks.io/documentation/using/monitoring/#benchmark-suite and within the recent [Microcks 1.9.0 release notes](https://microcks.io/blog/microcks-1.9.0-release/). Please let us know what you think about it!
